notify/templates.go

```diff
@@ -18,37 +17,37 @@ import (
 
 type TestTemplatesConfigBodyParams struct {
 	// Alerts to use as data when testing the template.
-	Alerts []*PostableAlert `json:"alerts"`
+	Alerts []*PostableAlert
```

Comment:
Ideally, I think we should accept []*types.Alert here because PostableAlert is API model in Alertmanager

Reply:
Sounds good, I'll move `OpenAPIAlertsToAlerts` conversion to `grafana/grafana` side

Reply:
please make an alias for it in alerting/notify package `OpenAPIAlertsToAlerts`